cg/meta/orders/case_submitter.py

```diff
@@ -67,7 +67,9 @@ def _validate_samples_available_to_customer(
             if not sample.internal_id:
                 continue
 
-            existing_sample: models.Sample = self.status.sample(sample.internal_id)
+            existing_sample: models.Sample = self.status.get_sample_by_internal_id(
```

Comment:
import `Samples`directly, not from `models`

Reply:
done